modules/flist/flist.go

```diff
@@ -94,7 +107,7 @@ func (f *flistModule) Mount(url, storage string) (string, error) {
 	logPath := filepath.Join(f.log, rnd) + ".log"
 
 	args := []string{
-		"-backend", filepath.Join(f.root, "backend", rnd),
+		"-backend", filepath.Join(path, "backend"),
```

Comment:
I think this can be simplified to this.
```suggestion
		"-backend", path,
```

Reply:
indeed